store/src/java/com/zimbra/cs/service/mail/ItemAction.java

```diff
@@ -99,13 +100,18 @@ public Element handle(Element request, Map<String, Object> context) throws Servi
 
         Element action = request.getElement(MailConstants.E_ACTION);
         String operation = action.getAttribute(MailConstants.A_OPERATION).toLowerCase();
-
-        String successes = handleCommon(context, request, operation, MailItem.Type.UNKNOWN);
+        boolean nonExistentIdsRequested = action.getAttribute(MailConstants.A_NON_EXISTENT_IDS).equals("1");
```

Comment:
Both "1" and "true" are valid values to request them.  Use the correct Element method to cope with that.